doorstop/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from doorstop.common import DoorstopError, DoorstopWarning, DoorstopInfo
 from doorstop.core import Item, Document, Tree
-from doorstop.core import build, find_document, find_item
```

Comment:
Some people might depend on these APIs, so if required, this will be a breaking change.

Reply:
I really doubt that any body would be relying on those methods.. From what I saw they were used by unit test only... they were providing a static and shared tree to all unit tests... With little change they were then no more needed so I removed them.

**Pro:**
* No static shared variables
* Do not keep code which is not needed

Reply:
Regardless, they were part of the public API. It's not big deal, but this change will have to be part of the next major release.